### PR TITLE
Fix external freetype that provides freetype-config.cmake

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@mathstuf @johnwparent Please review

**Comment by danlipsa** [comment]
@kwryankrattiger @vicentebolea Please review

**Review by kwryankrattiger — DISMISSED** [review]
LGTM

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -273,6 +273,9 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     patch("vtk-xdmf2-hdf51.13.1.patch", when="@5.10.0:5.10")
     patch("vtk-xdmf2-hdf51.13.2.patch", when="@5.10:")
 
+    # Fix VTK to work with external freetype using CONFIG mode for find_package
+    patch("FindFreeType.cmake.patch", when="@5.10.1:")
```
Comment (kwryankrattiger) [inline var/spack/repos/builtin/packages/paraview/package.py]: typo here, `Type` -> `type`

Comment (danlipsa) [reply var/spack/repos/builtin/packages/paraview/package.py]: Tried this on windows, so that did not matter. :-) Thanks!

**Review by kwryankrattiger — APPROVED** [review]

**Comment by mathstuf** [comment]
FYI, this patch should be sent to VTK itself as well.

**Pull request merged**